app/src/main/java/com/standuptracker/MainActivity.kt

```diff
@@ -14,38 +14,40 @@ import com.google.android.material.floatingactionbutton.FloatingActionButton
 import com.google.android.material.navigation.NavigationView
 import com.google.android.material.snackbar.Snackbar
 
-class MainActivity : AppCompatActivity() {
-
+class MainActivity : AppCompatActivity()
+{
```

Comment:
Where to put curlies used to be a big debate.  Best not to change this, as it is likely to draw the ire of the person who put them in the original location, and now sees that they are moved.  What Android Studio uses now (the way this used to be) is considered the default.

Reply:
Also, changing curlies makes comparisons difficult in GitHub, as it shows all of the changes.